apps/meteor/tests/end-to-end/api/teams.ts

```diff
@@ -1926,6 +1990,33 @@ describe('[Teams]', () => {
 			Promise.all([...[testTeamName, testTeamName2, teamName3].map((name) => deleteTeam(credentials, name)), deleteUser(unauthorizedUser)]),
 		);
 
+		it('should update sidepanel items to channels and discussions', async () => {
+			const sidepanelItems = ['channels', 'discussions'];
+			const response = await request
+				.post(api('rooms.saveRoomSettings'))
+				.set(credentials)
+				.send({
+					rid: testTeam.roomId,
+					sidepanel: { items: sidepanelItems },
+				});
+
+			expect('Content-Type', 'application/json');
+			expect(200);
+			expect(response.body).to.have.property('success', true);
```

Comment:
I believe you meant to use `teams.update` here, didn't you? :thinking: 
It'd be good to have tests for when `teams.update` and `teams.create` endpoints should throw errors here too

Reply:
actually no, I moved this test to `apps/meteor/tests/end-to-end/api/rooms.ts` but forgot to delete it here.
why: teams.update only refers to `teams` properties: name and type - which is not the case. `sidepanel` is being saved on the `room` object, therefore, using the `rooms.saveRoomSettings`
and there I added more tests to the error cases

Reply:
Got it, so teams.update would only be another way to trigger saveRoomSettings. We should be fine by covering tests on that side, then :+1: